plugins/imptcp/imptcp.c

```diff
@@ -765,17 +769,25 @@ AcceptConnReq(ptcplstn_t *pLstn, int *newSock, prop_t **peerName, prop_t **peerI
 		 */
 		sockflags = fcntl(iNewSock, F_SETFL, sockflags);
 	}
+
 	if(sockflags == -1) {
 		DBGPRINTF("error %d setting fcntl(O_NONBLOCK) on tcp socket %d", errno, iNewSock);
 		prop.Destruct(peerName);
 		prop.Destruct(peerIP);
 		ABORT_FINALIZE(RS_RET_IO_ERROR);
 	}
-
+	DBGPRINTF("pascal: %d\n", pLstn->pSrv->bEmitMsgOnOpen);
+	if(pLstn->pSrv->bEmitMsgOnOpen) {
+		LogMsg(0, RS_RET_NO_ERRCODE, LOG_INFO, "imptcp: connection established");
```

Comment:
The message should include the host that connected (name or IP), at least if possible at this stage. If not, please explain.